### PR TITLE
add Doxygen build dep to libheif, and drop libheif and libde265 to GCCcore

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/libheif/libheif-1.16.2-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -8,14 +8,16 @@ version = '1.16.2'
...
 
-toolchain = {'name': 'GCC', 'version': '11.3.0'}
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
 
 source_urls = ['https://github.com/strukturag/libheif/releases/download/v%(version)s/']
 sources = [SOURCE_TAR_GZ]
 checksums = ['7f97e4205c0bd9f9b8560536c8bd2e841d1c9a6d610401eb3eb87ed9cdfe78ea']
 
 builddependencies = [
+    ('binutils', '2.3'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/libheif/libheif-1.16.2-GCCcore-11.3.0.eb:18]:
```suggestion
    ('binutils', '2.38'),
```

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/c69fb77425dd83fd0ab6f483f143fa83 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Pull request merged**